app/models/item_order.rb

```diff
@@ -24,7 +20,7 @@ class ItemOrder < ApplicationRecord
   def total_price
     total_price = []
     user = User.find(user_id)
-    user.item_orders.map { |cart| total_price << cart.item.price * cart.quantity }
+    user.item_orders.map { |cart| total_price << cart.item.price }
```

Comment:
You could use `pluck` here instead of mapping, resulting in a performances improvement